STYLEGUIDE.adoc

```diff
@@ -240,6 +240,23 @@ Use Text instead of String unless you have a good reason not to.
 
 It’s 2022, use a proper, unicode aware string type instead of a linked list.
 
+=== Derive `Show`, define `Pretty`, use `Pretty` for human-readable output
+
+Always derive `Show`, do not define it manually.
```

Comment:
What about exception types? `Show` is hardcoded into `Exception`, but we'd like exception messages to be readable.

Reply:
I thought you could just define `displayException`? https://hackage.haskell.org/package/base-4.16.1.0/docs/Control-Exception.html#v:displayException

Reply:
Worth including, though.